src/Http/Authentication.Core/test/AuthenticationPropertiesTests.cs

```diff
@@ -302,6 +303,42 @@ public void GetBool()
             Assert.Equal("BAR", props.Items["foo"]);
         }
 
+        [Fact]
+        public void Roundtrip_Serializes_With_SystemTextJson()
+        {
+            var props = new AuthenticationProperties()
+            {
+                AllowRefresh = true,
+                ExpiresUtc = new DateTimeOffset(2021, 03, 28, 13, 47, 00, TimeSpan.Zero),
+                IssuedUtc = new DateTimeOffset(2021, 03, 28, 12, 47, 00, TimeSpan.Zero),
+                IsPersistent = true,
+                RedirectUri = "/foo/bar"
+            };
+
+            props.Items.Add("foo", "bar");
+
+            props.Parameters.Add("baz", "quux");
+
+            var json = JsonSerializer.Serialize(props);
```

Comment:
Can you give a quick example of the serialized output? I want to make sure it doesn't contain the Parameters collection. This test only verifies the that the Parameters are not deserialized.

Reply:
Sure - will post something shortly.

Reply:
Actually, we should take this further and avoid serializing any of the properties, only the Items.

Reply:
#31414